src/ice_ridge.F90

```diff
@@ -43,9 +45,12 @@ module ice_ridging_mod
 
 type, public :: ice_ridging_CS ; private
   logical :: &
-       new_rdg_partic = .false., & !< .true. = new participation, .false. = Thorndike et al 75
-       new_rdg_redist = .false.    !< .true. = new redistribution, .false. = Hibler 80
+  new_rdg_partic = .false., & !< .true. = new participation, .false. = Thorndike et al 75
+  new_rdg_redist = .false.    !< .true. = new redistribution, .false. = Hibler 80
   real :: mu_rdg = 3.0 !< e-folding scale of ridged ice, new_rdg_partic (m^0.5)
+  real :: area_underflow = 0.0 ! a non-dimesional fractional area underflow limit for the sea-ice
```

Comment:
@MJHarrison-GFDL Could you rewrite this in doxygen syntax?

Reply:
Actually these indentations also need to be fixed.  ("More than 2 after line continuation")